projects/plugins/jetpack/extensions/blocks/button/save.js

```diff
@@ -34,20 +35,25 @@ export default function ButtonSave( { attributes, blockName, uniqueId } ) {
 		[ `wp-block-jetpack-${ blockName }` ]: blockName,
 	} );
 
-	const buttonClasses = classnames( 'wp-block-button__link', {
-		'has-text-color': textColor || customTextColor,
-		[ textClass ]: textClass,
-		'has-background': backgroundColor || gradient || customBackgroundColor || customGradient,
-		[ backgroundClass ]: backgroundClass,
-		[ gradientClass ]: gradientClass,
-		'no-border-radius': 0 === borderRadius,
-		'has-custom-width': !! width,
-	} );
+	const buttonClasses = classnames(
+		'wp-block-button__link',
+		{
+			'has-text-color': textColor || customTextColor,
+			[ textClass ]: textClass,
+			'has-background': backgroundColor || gradient || customBackgroundColor || customGradient,
+			[ backgroundClass ]: backgroundClass,
+			[ gradientClass ]: gradientClass,
+			'no-border-radius': 0 === borderRadius,
+			'has-custom-width': !! width,
+		},
+		fontSize && { [ `has-${ fontSize }-font-size` ]: true }
```

Comment:
Same as https://github.com/Automattic/jetpack/pull/26108#discussion_r970549610, this one can be simplified a little bit as well

Reply:
Simplified using the same approach as the one suggested.